setup.sh

```diff
@@ -11,6 +11,24 @@ fi
 ${VENV_BINDIR}/pip3 install pip --upgrade
 ${VENV_BINDIR}/pip3 install -r requirements.txt
 
-if ! command -v docker &> /dev/null; then
-    printf "\n${HIGHLIGHT}Please install Docker${NO_COLOR}"
+exit_value=0
+
+if docker -v > /dev/null 2>&1; then
+    :
+else
+    echo "${HIGHLIGHT}Please install docker${NO_COLOR}"
+    exit_value=$(($exit_value+1))
+fi
+
+if chromedriver -v > /dev/null 2>&1; then
+    :
+else
+    echo "${HIGHLIGHT}Please install chromedriver${NO_COLOR}"
```

Comment:
What's `chromedriver`?  Can it be installed via `pip`?  `Docker` is pretty standard, `chromedriver` not so much.

Reply:
Maybe a comment on the readme?

Reply:
`chromedriver`'s what you need to run Selenium with Chrome.  You can install it with `pip`, but it yells at you when you do that in a venv because it's not a python library but an external application.  But yeah I'll clarify with instructions.